core/templates/dev/head/services/searchService.js

```diff
@@ -64,18 +64,41 @@ oppia.factory('searchService', [
     return _searchCursor === null;
   };
 
+  var updateSearchFields = function(itemsType, urlComponent,
+                                    selectionDetails) {
+    var itemCodeGroup = urlComponent.match(/=\("[A-Za-z%20" ]+"\)/);
+
+    if (itemCodeGroup == undefined) {
+      alert('Invalid search query url!');
```

Comment:
throw Error('Invalid search query url fragment for ' + itemsType + ': ' + urlComponent);